### ISSUE TITLE
Graphics module missing alt text

### ISSUE THREAD

**Issue by postmastersteve** [downshiftorg/prophoto7-issues#303]
This Github issue is synchronized with Zendesk:

**Ticket ID:** [#182017](https://prophotoblogs.zendesk.com/agent/tickets/182017)
**Priority:** normal
**Group:** Developers
**Requester:** josh@pro.photo
**Organization:** DownShift
**Issue escalated by:** Josh Arens

**Original ticket description:**

<blockquote><div class="zd-comment">In p6 the graphics module images could have alt text. That option appears to be missing. Was that intentional? And if so, is the info there and could be added later? <br>

<span class="collapse-signature"></span><div class="signature">
<p>Josh Arens<br>
ProPhoto Support</p>
<p>Follow us for the latest ProPhoto news and updates:<br>
<a href="https://pro.photo/blog/" rel="noreferrer">https://pro.photo/blog/</a></p>
</div>
</div></blockquote>

**Comment by danjonlam** [comment]
+1 Guy Milnes ticket 186638

**Comment by benjamintennant** [comment]
It seems to be present for me. Was this a change?

**Comment by postmastersteve** [comment]
+1 Hayley L Adhofer ticket 193196
